src/omnisharp/server.ts

```diff
@@ -253,63 +268,72 @@ export abstract class OmnisharpServer {
 
 	// --- start, stop, and connect
 
-	public start(solutionPath: string): Promise<void> {
-		if (!this._start) {
-			this._start = this._doStart(solutionPath);
+	private _start(launchTarget: LaunchTarget): Promise<void> {
+		const options = this._readOptions();
+		
+		let flavor: omnisharp.Flavor;
+		if (options.path !== undefined && options.usesMono === true) {
```

Comment:
If I am reading this correctly, one can use the 'usesMono' option to force the use of mono OmniSharp, but we don't have a corresponding option to force the use of CoreCLR omnisharp for folks who want to use CoreCLR omnisharp with a .csproj. Do I have that right?

If so, would it make sense to also have a way to force CoreCLR for folks dealing with PCL projects targeting CoreCLR?